src/equations/laplace_diffusion_2d.jl

```diff
@@ -28,8 +28,8 @@ end
 # TODO: parabolic; should this remain in the equations file, be moved to solvers, or live in the elixir?
 # The penalization depends on the solver, but also depends explicitly on physical parameters,
 # and would probably need to be specialized for every different equation.
-function penalty(u_outer, u_inner, inv_h, equations_parabolic::LaplaceDiffusion2D, dg::ViscousFormulationLocalDG)
-  return dg.penalty_parameter * (u_outer - u_inner) * equations_parabolic.diffusivity * inv_h
```

Comment:
To me, it *should* have been a breaking change, but by luck it doesn't seem to be?

Reply:
In any case, if we are uncertain about this, I can always remove `inv_h` from `cache_parabolic`, leave it in `penalty(...)` to make this a non-breaking PR, and then combine the maybe-breaking change of removing `inv_h` from `penalty` with https://github.com/trixi-framework/Trixi.jl/pull/1409.

Reply:
I've reverted the removal of `inv_h` in d1a42f2569d3cbf40a1d6d49a0e4c0de0836338f. This should make it so that this PR doesn't contain any breaking changes.

Reply:
Thanks!